applications/StructuralMechanicsApplication/custom_elements/solid_elements/base_solid_element.cpp

```diff
@@ -747,6 +747,7 @@ void BaseSolidElement::CalculateOnIntegrationPoints(
 
     if ( rOutput.size() != number_of_integration_points )
         rOutput.resize( number_of_integration_points, false );
+    rOutput.assign(number_of_integration_points, 0.0);
```

Comment:
?, when you do resize does not assign the value you give?

Reply:
Otherwise maybe remove the false

Reply:
apparently not. Currently you get random numbers in the posts

Reply:
Okay I just checked, the resize only assigns value to the new elements

Reply:
Just for you to have an example. If I have 2 materials (elastic and damage) and I want to print `DAMAGE`, the elastic zone plots random values :S

Reply:
I would remove the false is misleading

Reply:
Just noting that `false` is to say that previous values are not kept when resizing but doesn't call the initializer of the stored type.